extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -148,7 +149,8 @@ export abstract class DacFxConfigPage extends BasePage {
 	protected generateFilePath(): string {
 		let now = new Date();
 		let datetime = now.getFullYear() + '-' + (now.getMonth() + 1) + '-' + now.getDate() + '-' + now.getHours() + '-' + now.getMinutes();
-		return path.join(os.homedir(), this.model.database + '-' + datetime + this.fileExtension);
+		let rootPath = vscode.workspace.rootPath ? vscode.workspace.rootPath : os.homedir();
+		return path.join(rootPath, this.model.database + '-' + datetime + this.fileExtension);
```

Comment:
Can the database name contain illegal file characters?

Reply:
Opened an issue #5216. Will fix in a separate PR